packages/trpc/src/procedures/auth.ts

```diff
@@ -0,0 +1,161 @@
+import { z } from "zod";
+import { baseProcedureBuilder } from "../internal/init";
+const bcrypt = require('bcryptjs');
```

Comment:
We cannot use 'require' statements in ESM modules (import syntax). Also, bcrypt will need to be added as a dependency to this package